examples/hosts

```diff
@@ -1,14 +1,15 @@
 [lemmy]
 # to get started, copy this file to `inventory` and adjust the values below.
 # - `myuser@example.com`: replace with the destination you use to connect to your server via ssh
+# - `ansible_user=root`: replace `root` with your the username you use to connect to your ssh server
 # - `domain=example.com`: replace `example.com` with your lemmy domain
 # - `letsencrypt_contact_email=your@email.com` replace `your@email.com` with your email address,
 #                                              to get notifications if your ssl cert expires
 # - `lemmy_base_dir=/srv/lemmy`: the location on the server where lemmy can be installed, can be any folder
 #                                if you are upgrading from a previous version, set this to `/lemmy`
 # - `lemmy_version`: <Optional> The back end version.
 # - `lemmy_ui_version`: <Optional> overrides the front end version.
-myuser@example.com  domain=example.com  letsencrypt_contact_email=your@email.com  lemmy_base_dir=/srv/lemmy
+example.com  ansible_user=root domain=example.com  letsencrypt_contact_email=your@email.com  lemmy_base_dir=/srv/lemmy
```

Comment:
where `{{ domain }}` would be the first `example.com`

But this can be anything if you have your own `.ssh/config`

Reply:
I think it serves its purpose as an example.